config.xml

```diff
@@ -36,4 +36,16 @@
     <plugin name="cordova-plugin-console" spec="1.0.3" />
     <plugin name="fh-cordova-plugin-device" spec="https://github.com/feedhenry/fh-cordova-plugin-device.git#fh0.0.4" />
     <plugin name="cordova-plugin-whitelist" spec="1.2.2" />
+    <edit-config target="NSCameraUsageDescription" file="*-Info.plist" mode="merge">
+        <string>need camera access to take pictures</string>
+    </edit-config>
+    <edit-config target="NSPhotoLibraryUsageDescription" file="*-Info.plist" mode="merge">
+        <string>need to photo library access to get pictures from there</string>
```

Comment:
language grammar - just change from `need to photo library access to get pictures from there`to `need to access photo library to get pictures from there`